src/Compilers/CSharp/CscCore/CscCore.csproj

```diff
@@ -16,7 +16,7 @@
     <AutoGenerateBindingRedirects>True</AutoGenerateBindingRedirects>
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <TargetFramework>netcoreapp1.1</TargetFramework>
-    <RuntimeIdentifiers>win7-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;osx.10.12-x64</RuntimeIdentifiers>
+    <RuntimeIdentifiers>win7-x64;$(UnixRuntimeIdentifier)</RuntimeIdentifiers>
```

Comment:
On windows, I ran Restore.cmd with my changes and then opened Roslyn.sln with `Version 15.2 (26424.2-Preview) d15rel Internal Preview - Dogfood` which is the newest build I had available.  The solution builds fine, IntelliSense works for both types defined in Roslyn and types that are coming in via packages (e.g. I can see members on string, while editing code in CscCore.csproj).  Go to definition also works.  Is there something else you'd like me to check?